main.tf

```diff
@@ -19,6 +19,15 @@ data "azurerm_subnet" "subnet" {
   resource_group_name  = var.vnet_resource_group_name
 }
 
+locals {
+  # umi == user managed identity, smi == system managed identity
+	use_umi = length(var.linux_agents_configuration.user_assigned_identity_ids) > 0
+  use_smi = var.linux_agents_configuration.use_system_assigned_identity
```

Comment:
run terraform fmt on file

Reply:
done in ./ and other test folders. cf commit bafabf931449ad3bdafd0fa30e5e77cb1447399c